backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java

```diff
@@ -114,48 +120,69 @@ public PatientBulkUploadResponse processPersonCSV(InputStream csvStream, UUID fa
             extractedData.getFirstName().getValue(),
             extractedData.getLastName().getValue(),
             parseUserShortDate(extractedData.getDateOfBirth().getValue()),
-            org,
-            facility)) {
+            currentOrganization,
+            assignedFacility)) {
           continue;
         }
 
-        _personService.addPatient(
-            facilityId,
-            null, // lookupID
-            extractedData.getFirstName().getValue(),
-            extractedData.getMiddleName().getValue(),
-            extractedData.getLastName().getValue(),
-            extractedData.getSuffix().getValue(),
-            parseUserShortDate(extractedData.getDateOfBirth().getValue()),
-            address,
-            country,
-            List.of(
-                new PhoneNumber(
-                    parsePhoneType(extractedData.getPhoneNumberType().getValue()),
-                    extractedData.getPhoneNumber().getValue())),
-            parsePersonRole(extractedData.getRole().getValue(), false),
-            List.of(extractedData.getEmail().getValue()),
-            convertRaceToDatabaseValue(extractedData.getRace().getValue()),
-            convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()),
-            null,
-            convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()),
-            parseYesNo(extractedData.getResidentCongregateSetting().getValue()),
-            parseYesNo(extractedData.getEmployedInHealthcare().getValue()),
-            null,
-            null);
+        // create new person with current organization, then add to new patients list
+        Person newPatient =
+            new Person(
+                currentOrganization,
+                assignedFacility,
+                null,
```

Comment:
nit: could you mark out what all the various `nulls` in the constructor are with comments, like we had with `lookupId`?

Reply:
I thought these weren't needed b/c intelliJ tells me what type they are 😂 but for readability outside of intelliJ, then I agree with adding comments

Reply:
✅

Reply:
Yeah I thought about that right after I left the comment 😆 But I do think it helps for looking at the code elsewhere.